pages/api/dpu/aliases.ts

```diff
@@ -0,0 +1,30 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { saveUserAliasesToDb } from '../../../utils/db/aliases';
+
+const aliasesHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+    console.info("[newEndpointHandler] Processing new endpoint request...");
```

Comment:
??
Please reword this log message to make it useful during debugging.

Reply:
done